### PR TITLE
fixed menu

### PR DESCRIPTION
**Pull request by andjan3**

### CODE REVIEW

File: .gitignore
Comment (bastakom) [inline .gitignore:1]: remove line

File: components/ui/nav/nav.tsx [hunk 1]
```diff
@@ -10,18 +32,87 @@ interface NavBarProps {
       alt: string;
     };
     site_title: string;
+    Menu: LinkItem[];
```
Comment (bastakom) [inline components/ui/nav/nav.tsx:35]: Use small letters for data

File: package-lock.json
Comment (bastakom) [inline package-lock.json:1]: Remove package-lock.json

File: .gitignore
Comment (andjan3) [reply .gitignore:1]: Fixat!

**Review by bastakom — CHANGES_REQUESTED** [review]

File: components/ui/nav/nav.tsx (hunk 1 above)
Comment (andjan3) [reply components/ui/nav/nav.tsx]: Changed to camelCase, or did you mean to change it to all lowercase instead?

**Review by bastakom — APPROVED** [review]

**Pull request merged**